keanu-project/src/main/java/io/improbable/keanu/vertices/intgr/nonprobabilistic/CastIntegerVertex.java

```diff
@@ -11,28 +12,24 @@
 
 public class CastIntegerVertex extends IntegerVertex implements NonProbabilistic<IntegerTensor> {
```

Comment:
Are the names here appropriate do you think?  Should it be like CastToIntegerVertex ?  Would make more sense from a python PoV (ie CastToInteger(input_double_vertex))?

Reply:
The "To" does make it a bit clearer. Will change.